roles/provision-data-sync-apb/defaults/main.yml

```diff
@@ -25,3 +25,7 @@ proxy_version: "v1.1.0"
 dashboards_configmap: grafana-dashboards-configmap
 dashboard_filename: data-sync-dashboard.json # name the file will have inside the config map
 dashboard_file_contents: "{{ lookup('file','../files/data-sync-dashboard.json') }}" # Contents of the dashboard file
+
+# Security complexity limits
+complexity_limit: "{{ COMPLEXITY_LIMIT | default(10000) }}"
```

Comment:
As suggested we may move out this to use it in parameters

Reply:
What if user would delete the default value from the dialog? I'm not 100 % sure, but this could fail during provisioning since the env var will have undefined value.

Reply:
Actually in failed in CI: https://jenkins-wendy.ci.feedhenry.org/job/AeroGearCatalog/job/data-sync-apb/job/PR-23/5/console